faqs/common-errors-faq.md

```diff
@@ -77,7 +76,40 @@ An example: For `POST:/accesstoken/get` the error may be like this if you send a
 
 ## Why do I get `HTTP 401 Unauthorized`?
 
-This means you are using the wrong API keys or Authorization header.
+If you get errors like below from Microsoft Azure, this means you are using the wrong API keys or Authorization header:
```

Comment:
We prefer to not mention that we use azure, and it's often not azure that returns this response. Maybe reformulate the deleted line a little bit instead?